docs/data/base/components/badge/badge.md

```diff
@@ -59,17 +59,12 @@ The Badge component is composed of a root `<span>` that houses the element that
 </span>
 ```
 
-### Slot props
+### Custom structure
 
-:::info
-The following props are available on all non-utility Base components.
-See [Usage](/base/getting-started/usage/) for full details.
-:::
-
-Use the `component` prop to override the root slot with a custom element:
+Use the `slots.root` prop to override the root slot with a custom element:
```

Comment:
I'd remove this paragraph and change the next one to: "Use the `slots` prop to override the root or any other interior slot."